ce/api/data.py

```diff
@@ -124,6 +128,21 @@ def data(
                 climatological_statistic
             )
         )
+    if percentile is not None:
+        try:
+            percentile = float(percentile)
+        except ValueError:
+            raise Exception(
+                "Percentile parameter {} not convertable to a number".format(percentile)
+            )
```

Comment:
`Exception` is the most generic possible exception to raise. I think Python preferred practice is to be more specific, in this case maybe `ValueError`?